packages/requests/src/lib/requests-cache.ts

```diff
@@ -70,10 +84,17 @@ export function isRequestCached<S extends StateOf<typeof withRequestsCache>>(
   key: string | number | string[] | number[],
   options?: { value?: CacheState['value'] }
 ) {
+  const now = Date.now();
```

Comment:
move to the inner function

Reply:
moved to inner function, but I think there should still be a variable. otherwise we're calculating it for every iteration inside the 'some'.